accessible/tree.service.js

```diff
@@ -484,8 +484,9 @@ blocklyApp.TreeService = ng.core.Class({
         // Return the focus to the workspace tree containing the input field.
         document.getElementById(treeId).focus();
 
-        // Note that Tab events are allowed to propagate through.
-        if (e.keyCode == 27) {
+        // Note that Tab and Enter events stop propagating, this behavior is
```

Comment:
Just FYI, Tab events were originally allowed to propagate through because, in addition to closing the input field, the focus would _also_ move to the next/previous active element in the DOM as needed. If we stopped them here, that wouldn't happen.